[strings] some Hindi translations

j13m126 opened the pull request:
I used wikidata to translate a few strings into Hindi. 

Here is the query for [wikidata query service](https://query.wikidata.org/) I used to get the country names:

`SELECT ?country ?name_en ?name_hi
WHERE {
  ?country wdt:P31 wd:Q3624078. 
  ?country rdfs:label ?name_en.
  ?country rdfs:label ?name_hi.
  FILTER (LANG(?name_en) = 'en')
  FILTER (LANG(?name_hi) = 'hi')
}`

For the states of India I replaced Q3624078 with Q12443800 in the query above.

For the amenites, shops and cuisines ([d70afd0](https://github.com/organicmaps/organicmaps/pull/4266/commits/d70afd0834bf0b06656667cc30b5138cbcc38ba3)) I used Google Translate exclusively, wikidata wasn't helpful here.

Signed-off-by: Lukas Kronberger <j13m126@posteo.de>

biodranik reviewed (approved):
Thanks! There are also data/strings/sounds.txt and iphone/plist.txt

biodranik commented:
@j13m126 do you _read_ Hindi?

j13m126 commented:
I wrote a post on [reddit](https://www.reddit.com/r/openstreetmap/comments/10qt1py/proofreading_hindi_translations_for_organic_maps/) asking for help with proofreading and, thanks to Pidwid, already got some [feedback](https://docs.google.com/spreadsheets/d/1pye9Mv7JZlnRL1uaXDvJmLwTSsnBT1DTgXp80FprHKw/edit?usp=sharing) on shops. He said he will do amenities soon.

j13m126 commented:
> > I wrote a post on [reddit](https://www.reddit.com/r/openstreetmap/comments/10qt1py/proofreading_hindi_translations_for_organic_maps/) asking for help with proofreading and, thanks to Pidwid, already got some [feedback](https://docs.google.com/spreadsheets/d/1pye9Mv7JZlnRL1uaXDvJmLwTSsnBT1DTgXp80FprHKw/edit?usp=sharing) on shops. He said he will do amenities soon.
> 
> Requested access, thanks! Could you please add Suggestions to this PR?
> 
> <img alt="image" width="909" src="https://user-images.githubusercontent.com/1799054/216755713-f683ffe1-59b5-4bd6-bfef-e12ced9368f7.png">

Access should work now. Will add suggestions in the evening.

Saurmandal commented on data/categories.txt:
```diff
@@ -758,6 +760,7 @@ nl:2Bar|2pub|kroeg|bier|drinken
 fi:2Baari|pubi
 fr:2Bar|2pub|bière|boisson
 de:2Bar|2Pub|4Kneipe|Bier|Trinken|4Gaststätte|4Bars und Kneipen|Brauhaus|Cocktail-Lounge
+hi:मधुशाला|सराय
```
Add "पब" as another alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -1016,6 +1021,7 @@ nl:4Schoonheidsmiddelen
 fi:4Kosmetiikka|kauneudenhoito
 fr:4Produits de beauté|soins de beauté
 de:4Kosmetik|Schönheitspflege
+hi:सौंदर्य प्रसाधन की दुकान
```
Add "कॉस्मेटिक्स" as another alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -1163,13 +1173,15 @@ shop-grocery|@food
 en:Health food
 de:4Reformhaus|5Naturkostladen|4Bioladen
 et:Tervisetoit
+hi:हेल्थ फ़ूड स्टोर
```
Add "स्वस्थ भोजन की दुकान" as primary alias.

j13m126 replied:
there is no ranking in search words, as far as I know. Do you want your suggestion to be visible when you click on a point of interest?

Saurmandal replied:
Yea

Saurmandal commented on data/categories.txt:
```diff
@@ -1552,6 +1573,7 @@ nl:4Juwelier
 fi:Korukauppa
 fr:4Bijouterie
 de:4Juwelier|Juweliergeschäft|Schmuck
+hi:आभूषण की दुकान
```
Add "ज्वैलरी" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -1589,6 +1611,7 @@ nl:4Opticien
 fi:4Optikko
 fr:4Opticien
 de:4Optiker|5Brillengeschäft|5Augenoptiker
+hi:प्रकाशविज्ञानशास्री
```
Add "ऑप्टिशियन" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -1885,6 +1913,7 @@ nl:Drank|slijterij
 fi:Juomat
 fr:4Boissons
 de:5Getränkemarkt
+hi:पेय की दुकान
```
Replace with "पेय पदार्थ".

Saurmandal commented on data/categories.txt:
```diff
@@ -2705,6 +2749,7 @@ nl:3Picknickplaats|picknickweide|picknicktafel
 fi:3Piknikpaikat
 fr:3Terrain de pique-nique|3aire de pique-nique
 de:3Picknickplatz|5Grillplatz
+hi:बीबीक्यू
```
Replace with "बार्बीक्यू".

Saurmandal commented on data/categories.txt:
```diff
@@ -4049,6 +4105,7 @@ nl:4Fietsverhuur|2huur
 fi:Polkupyöränvuokraus
 fr:Location de vélos|vélo|bicyclette|3location
 de:4Fahrradverleih|Fahrrad|Fahrradvermietung|Radfahren|Radverleih|Velo|3Verleih
+hi:साइकिल किराए पर लेना
```
Replace with "किराए पर साइकिल".

Saurmandal commented on data/categories.txt:
```diff
@@ -4113,6 +4171,7 @@ et:Auto jagamine
 eu:Partekatu autoa
 fi:Autojen yhteiskäyttö
 fr:Station d'autopartage|auto-partage
+hi:कार साझा करना
```
Replace with "गाड़ी साँझा".

Saurmandal commented on data/categories.txt:
```diff
@@ -4146,6 +4205,7 @@ nl:3Autoverhuur|4huurauto|auto delen|auto|huur
 fi:4Autovuokraamo
 fr:4Location de voitures|auto|loueur|location|partage
 de:3Autoverleih|Mietauto|Auto|Fahrzeug Mieten|Verleih|KFZ-Vermietung|Fahrzeugvermietung
+hi:किराए पर कार लेना
```
Replace with "किराए पर गाड़ियाँ".

Saurmandal commented on data/categories.txt:
```diff
@@ -4226,6 +4287,7 @@ fa:ﮓﻨﯿﻟﻮﺑ ﻦﻟﺎﺳ
 fi:Keilarata
 fr:Salle de bowling
 he:גנילואב םלוא
+hi:बोलिंग एले
```
Replace with "बोलिंग ऐली".

Saurmandal commented on data/categories.txt:
```diff
@@ -4994,6 +5062,7 @@ nl:3Apotheek
 fi:3Apteekki
 fr:3Pharmacie
 de:3Apotheke|Pharmazie
+hi:फार्मेसी
```
Add "दवाखाना" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -5101,6 +5170,7 @@ nl:3Brievenbus|post
 fi:Postilaatikko
 fr:3Boîte aux lettres|courrier
 de:3Briefkasten|3Postfach|post
+hi:पत्र पात्र
```
Replace with "पोस्ट बॉक्स".

Saurmandal commented on data/categories.txt:
```diff
@@ -9968,7 +10050,7 @@ nl:3Internet
 fr:3Internet
 de:3Internet|Internetzugang
-hi:अंतरजाल
+hi:अंतरजाल|बेतार इंटरनेट पहुंच
```
Replace with "इंटरनेट उपलब्ध".

j13m126 replied:
should it replace both search words?

Saurmandal replied:
I'd keep the second one tbf

Saurmandal commented on data/categories.txt:
```diff
@@ -10133,6 +10215,7 @@ nl:4Copy shop|Kopieerwinkel|printer|4drukker
 fi:5Painotalo|5Tulostaja
 fr:Boutique de photocopies|5Imprimerie
 de:4Kopierladen|4Drucker|Copyshop
+hi:कॉपी की दुकान
```
Replace with "फ़ोटोकॉपी की दुकान".

Saurmandal commented on data/categories.txt:
```diff
@@ -10204,6 +10288,7 @@ nl:Reisbureau|reisagentschap|reizen
 fi:Matkatoimisto
 fr:Agence de voyages|Voyages organisés
 de:5Reisebüro|Reisen|Rundreisen|Reisevermittlung|Reisevermittler|Reiseagentur|Touren|Ausflüge|Urlaub|Touristeninformation|Last-Minute-Tour
+hi:ट्रैवल एजेंसी
```
Add "यात्रा एजेंसी" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -10275,6 +10361,7 @@ nl:Stomerij|chemisch reinigen|droogkuis|wassen|stomen
 fi:Kuivapesula|pesula
 fr:Nettoyage à sec|Pressing|Lavage
 de:Chemische Reinigung|4Reinigung
+hi:सूखी धुलाई
```
Add "ड्राय वॉश" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -10312,6 +10399,7 @@ nl:Bandenwinkel
 fi:Rengasliike
 fr:Magasin de pneus
 de:Reifen-Shop|4Autoreifen|Reifen
+hi:टायर की दुकान
```
Replace with "पहियों की दुकान".

Saurmandal commented on data/categories.txt:
```diff
@@ -10683,6 +10776,7 @@ nl:Bandenreparatie
 fi:Rengashuolto
 fr:Réparation de pneus
 de:Reifenservice|Reifen|6Reifenreparatur|Reifeninstandsetzung|Reifenpannenreparatur|Reifenpannenbehebung|Reifenersatz|Pneureparatur|Pneu|Pannenreparatur
+hi:टायर की मरम्मत
```
Replace with "पहियों की मरम्मत".

Saurmandal commented on data/categories.txt:
```diff
@@ -11007,6 +11106,7 @@ nl:Drankautomaat
 fi:Juoma-automaatti
 fr:Distributeur de boissons|cannettes
 de:9Getränkeautomat
+hi:पेय डिस्पेंसर
```
Replace with "पेय पदार्थ डिस्पेंसर".

Saurmandal commented on data/categories.txt:
```diff
@@ -11062,6 +11163,7 @@ eu:4Aparkatzeko txartelak ordaintzeko makina
 fa:پارکو متر
 fi:Pysäköintimaksuautomaatti
 fr:Horodateur
+hi:पार्किंग के टिकिट
```
Replace with "पार्किंग टिकट".

Saurmandal commented on data/categories.txt:
```diff
@@ -11338,6 +11446,7 @@ nl:5Bookmaker
 fi:Kustantaja
 fr:5Bookmaker
 de:5Buchmacher|3Wettbüro
+hi:बूक मेय्केड़
```
Replace with "बुकी".

Saurmandal commented on data/categories.txt:
```diff
@@ -12398,6 +12514,7 @@ et:5Õlleaed
 eu:5Biergarten
 fi:5Biergarten
 fr:5Biergarten
+hi:भालू बगीचा
```
Replace with "बीयर की दुकान का बगीचा".

Saurmandal commented on data/categories.txt:
```diff
@@ -12771,6 +12894,7 @@ et:Dušš
 eu:Dutxa
 fi:Suihku
 fr:Douche
+hi:बौछार
```
Replace with "शावर".

Saurmandal commented on data/categories.txt:
```diff
@@ -12809,6 +12933,7 @@ et:4Defibrillaator
 eu:Desfibriladorea
 fi:4Defibrillaattori
 fr:4Défibrillateur|DAE
+hi:वितंतुविकंपनित्र
```
Add "डीफ़िब्रिलिएटर" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -12847,6 +12972,7 @@ et:Tuletõrjehüdrant
 eu:Suteen hidrantea
 fi:5Paloposti
 fr:Bouche d’incendie|borne d'incendie
+hi:अग्नि हाईड्रेंट
```
Replace with "फ़ायर हाइड्रेंट".

Saurmandal commented on data/categories.txt:
```diff
@@ -13750,6 +13889,7 @@ en:Money lender
 be:Ліхвяр
 de:4Geldverleiher
 et:Rahalaenutus
+hi:साहूकार
```
Add "हवलदार" as primary alias.

Saurmandal commented on data/categories.txt:
```diff
@@ -14318,6 +14473,7 @@ fa:ﯼﺯﺎﺳﺮﻄﻋ
 fi:Hajuvedet
 fr:Parfumerie
 he:תמַשְׂבָּ
+hi:गंध-द्रव्य
```
Replace with "पर्फ़्यूम्स की दुकान".

Saurmandal commented on data/categories.txt:
```diff
@@ -14394,6 +14551,7 @@ fa:ﺭﺎﺒﻧﺍ ﻩﺭﺎﺟﺍ
 fi:Varastoinnin vuokraus
 fr:Location de stockage
 he:ןוסחא תרכשה
+hi:भंडारण किराये
```
Replace with "किराए पर भंडार".

Saurmandal commented on data/categories.txt:
```diff
@@ -14508,6 +14668,7 @@ fa:ﺖﻋﺎﺳ
 fi:Kellot
 fr:Montres
 he:םינועש
+hi:घड़ियों
```
Replace with "घड़ियाँ".

Saurmandal commented on data/categories.txt:
```diff
@@ -14546,6 +14707,7 @@ fa:ﯽﺷﻭﺮﻓ ﻩﺪﻤﻋ ﻩﺎﮕﺷﻭﺮﻓ
 fi:Tukkukauppa
 fr:Magasin de gros
 he:תיאנוטיס תונח
+hi:थोक दुकान
```
Replace with "थोक सामान की दुकान".

Saurmandal commented on data/categories_cuisines.txt:
```diff
@@ -77,6 +79,7 @@ nl:Ontbijt
 fi:Aamiainen
 fr:Petit déjeuner
 de:Frühstück
+hi:सुबह का नाश्ता
```
Replace with "नाश्ता".

Saurmandal commented on data/categories_cuisines.txt:
```diff
@@ -139,6 +143,7 @@ en:Bubble Tea|boba tea
 ru:Чай с шариками
 es:Té de burbujas
 eu:Burbuila tea
+hi:बुलबुला चाय
```
Replace with "बुलबुले वाली चाय".

Saurmandal commented on data/categories_cuisines.txt:
```diff
@@ -358,6 +369,7 @@ nl:Frituur
 fi:Uppopaistettu
 fr:Friterie
 de:Friterie
+hi:तलने
```
Replace with "पकोड़े".

Saurmandal commented on data/categories_cuisines.txt:
```diff
@@ -429,6 +442,7 @@ eu:Txakur beroak
 fi:Hotdog
 fr:Hot-dog
 he:נקניקיה
+hi:हॉट - डॉग
```
Remove space on either side of hyphen.

Saurmandal reviewed (changes requested):
Proofread Hindi translations.

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -19203,6 +19560,7 @@
     eu = Apustu etxea
     fa = صحافی
     fi = Kustantaja
+    hi = बुकी
```
suggestion by pidwid: सट्टा लगाने की जगह

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -19305,6 +19665,7 @@
     fa = فروشگاه
     fi = Autokauppa
     fr = Concessionnaire automobile
+    hi = कार डीलरशिप
```
suggestion by pidwid: गाड़ी के विक्रेता

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -19599,6 +19968,7 @@
     fa = فروشگاه
     fi = Tietokonekauppa
     fr = Magasin d'informatique
+    hi = कंप्यूटर स्टोर
```
suggestion by pidwid: कंप्यूटर की दुकान

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -20024,6 +20405,7 @@
     fi = Maatilaruokakauppa
     fr = Magasin d'alimentation à la ferme
     he = הווחל ןוזמ תונח
+    hi = खेत भोजन की दुकान
```
suggestion by pidwid: खेती से उत्पन्न खाद्द की दुकान

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -20165,6 +20550,7 @@
     fa = فروشگاه
     fi = Taimitarha
     fr = Jardinerie
+    hi = बगीचे की दुकान
```
suggestion by pidwid: बगीचे की सामग्री की दुकान

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -20478,6 +20871,7 @@
     fa = دَکهِ
     fi = Kioski
     fr = Kiosque
+    hi = कीओस्क
```
suggestion by pidwid: गुमटी

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -21042,6 +21449,7 @@
     fa = فروشگاه
     fi = Valokuvakauppa
     fr = Matériel de photographie
+    hi = फोटोशॉप
```
suggestion by pidwid: छायाचित्रण की दुकान

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -21220,6 +21632,7 @@
     fa = لوازم التحریر
     fi = Kirjoitustarvikekauppa
     fr = Papeterie
+    hi = स्टेशनरी की दुकान
```
suggestion by pidwid: लेखन सामग्री की दुकान

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -21426,6 +21844,7 @@
     fa = اژانس مسافرتی
     fi = Matkatoimisto
     fr = Agence de voyages
+    hi = यात्रा एजेंसी
```
suggestion by pidwid: यात्रा अभिकरण

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -21496,6 +21916,7 @@
     fa = فروشگاه
     fi = Halpakauppa
     fr = Bazar
+    hi = वेराइटी स्टोर
```
suggestion by pidwid: छोटी वस्तुओं की सस्ती दूकान

j13m126 commented on data/strings/types_strings.txt:
```diff
@@ -22251,6 +22691,7 @@
     fi = Hajuvedet
     fr = Parfumerie
     he = תמַשְׂבָּ
+    hi = पर्फ़्यूम्स की दुकान
```
suggestion by pidwid: सुगंध सामग्री की दुकान

j13m126 reviewed (commented):
These are the changes suggested by [pidwid](https://www.reddit.com/user/pidwid). Some have already been corrected, so please let me know if I should add these suggestions to the search terms or also replace them. @Saurmandal

biodranik commented:
There is an easier way to proof-read and fix translations by anyone who does not know how to work with Github and add suggestions here.
You can download these text files and edit them manually, and then send them back to us in a .zip archive at translations@organicmaps.app, or attach them here into a comment:

1. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/categories.txt
2. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/categories_cuisines.txt
3. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/countries_names.txt
4. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/strings/strings.txt
5. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/strings/brands_strings.txt
6. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/strings/sound.txt
7. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/strings/types_strings.txt
8. https://raw.githubusercontent.com/organicmaps/organicmaps/b3e8c4e87ce9ef9c5d0dabb3b7b40a82cc8bc5bc/data/plist.txt

Please read the comments in each file and try to understand the context first. The language code for Hindi is `hi`. Try to follow the sort order when inserting the string. Please don't hesitate to ask if something is not clear. And thank you for your help and efforts! 🙏

proxygeek commented on data/categories_cuisines.txt:
```diff
@@ -256,6 +264,7 @@ nl:Diner
 fi:Lounaskahvila
 fr:Diner américain
 de:Diner
+hi:भोजनगृह
```
It should be just भोजन , not भोजनगृह which would mean a place for dinner. Not the "cuisine".

```suggestion
hi:भोजन
```

proxygeek commented on data/categories_cuisines.txt:
```diff
@@ -839,6 +863,7 @@ nl:Steakhouse
 fi:Pihviravintola
 fr:Grillades
 de:Steakhaus
+hi:स्टेक हाउस
```
```suggestion
hi:स्टेक (माँस का कबाब)
```

j13m126 replied:
@rtsisyk is it ok to use parenthesis here?

proxygeek commented on data/categories_cuisines.txt:
```diff
@@ -907,6 +933,7 @@ nl:Tapas
 fi:Tapas
 fr:Tapas
 de:Tapas
+hi:तपस
```
```suggestion
hi:तापा
```

proxygeek commented on iphone/plist.txt:
```diff
@@ -144,6 +147,7 @@
     fi = Tämänhetkisen sijainnin selvittäminen sovellusta ajettaessa taustalla on tärkeää sovelluksen täydellisen toiminnan takaamiseksi. Sijaintitietoja käytetään reittiopastukseen sekä kuljetun reitin tallennukseen.
     fr = La localisation actuelle en arrière-plan est nécessaire pour profiter pleinement de toutes les fonctionnalités de l'application. Elle est utilisée dans les options de navigation et d'enregistrement de votre chemin récemment parcouru.
     he = גילוי המיקום הנוכחי ברקע נחוץ כדי ליהנות באופן מלא מהפונקציונליות של האפליקציה. הוא משמש לניווט ולשמירה על המסלול שנסעת לאחרונה.
+    hi = ऐप की कार्यक्षमता का पूरी तरह से आनंद लेने के लिए पृष्ठभूमि में मार्ग का पता लगाना आवश्यक है। इसका उपयोग नेविगेशन और आपके हाल ही में यात्रा किए गए ट्रैक को सहेजने के लिए किया जाता है।
```
```suggestion
    hi = ऐप की कार्यक्षमता का पूरी तरह से आनंद लेने के लिए पृष्ठभूमि में स्थान का पता लगाना आवश्यक है। इसका उपयोग मार्गदर्शन और आपके हाल ही में किए गए यात्रा को सहेजने के लिए किया जाता है।
```

proxygeek commented on iphone/plist.txt:
```diff
@@ -185,6 +189,7 @@
     fi = Sijaintitietojasi käytetään reittivalintoihin ja viimeaikaisten matkojesi tallentamiseen.
     fr = La définition de lemplacement actuel est utilisée dans certaines fonctions comme le suivi dun trajet et la sauvegarde des trajets récemment effectués.
     he = קביעת המיקום שלך נחוצה לניווט ולשמירה על המסלול שנסעת לאחרונה.
+    hi = नेविगेशन के लिए और अपने हाल ही में यात्रा किए गए ट्रैक को सहेजने के लिए अपना स्थान निर्धारित करना आवश्यक है।
```
```suggestion
    hi = मार्गदर्शन के लिए और अपने हाल ही में किए गए यात्रा को सहेजने के लिए आपका स्थान निर्धारित करना आवश्यक है।
```

proxygeek commented on data/categories.txt:
```diff
@@ -521,6 +521,7 @@ he:כספומט
 sk:3Bankomat
 sw:Benki|fedha
 fa:خود پرداز
+hi:स्वचालित गणक मशीन
```
```suggestion
hi:स्वचालित गणक मशीन (एटीएम)
```

proxygeek commented on data/categories.txt:
```diff
@@ -757,6 +760,7 @@ nl:2Bar|2pub|kroeg|bier|drinken
 fi:2Baari|pubi
 fr:2Bar|2pub|bière|boisson
 de:2Bar|2Pub|4Kneipe|Bier|Trinken|4Gaststätte|4Bars und Kneipen|Brauhaus|Cocktail-Lounge
+hi:मधुशाला|सराय|पब
```
```suggestion
hi:मधुशाला|सराय|पब|शराब की दूकान
```

proxygeek commented on data/categories.txt:
```diff
@@ -2220,6 +2263,7 @@ nl:3Kiosk
 fi:3Kioski
 fr:3Kiosque
 de:3Kiosk
+hi:कीओस्क
```
```suggestion
hi:कीओस्क | खोका
```

proxygeek commented on data/categories.txt:
```diff
@@ -3179,6 +3244,7 @@ fa:نشانگر مرز
 fi:Rajamerkki
 fr:Borne frontière
 he:אבן גבול
+hi:सीमा का पत्थर
```
```suggestion
hi:ऐतिहासिक सीमा का पत्थर
```

proxygeek commented on data/categories.txt:
```diff
@@ -3559,6 +3628,7 @@ fa:گردشگری
 fi:Muistomerkki
 fr:Mémorial
 he:פסל
+hi:प्रतिमा
```
```suggestion
hi:स्मारक प्रतिमा
```

proxygeek commented on data/categories.txt:
```diff
@@ -4381,6 +4470,7 @@ nl:Hogeschool|hoger beroepsonderwijs|hbo
 fi:4College
 fr:Enseignement post-secondaire
 de:4Hochschule|4Fachschule|4Kolleg
+hi:महाविद्यालय
```
```suggestion
hi:महाविद्यालय|कॉलेज
```

proxygeek commented on data/categories.txt:
```diff
@@ -6571,6 +6704,7 @@ nl:Bos|woud
 fi:Metsä
 fr:Forêt|bois
 de:Wald|Urwald|Nadelwald|Laubwald|Mischwald|Forst
+hi:वन
```
```suggestion
hi:वन|जंगल
```

proxygeek commented on data/categories.txt:
```diff
@@ -9867,6 +10050,7 @@ nl:3Internet
 fi:3Internet
 fr:3Internet
 de:3Internet|Internetzugang
+hi:इंटरनेट उपलब्ध|बेतार इंटरनेट पहुंच
```
```suggestion
hi:इंटरनेट उपलब्ध|वाईफ़ाई उपलब्ध
```

proxygeek commented on data/categories.txt:
```diff
@@ -9904,6 +10088,7 @@ nl:Strand
 fi:Ranta
 fr:Plage
 de:Strand
+hi:बालूतट
```
```suggestion
hi:बालू तट
```

proxygeek commented on data/categories.txt:
```diff
@@ -13433,6 +13690,7 @@ et:3Geiser
 eu:3Geiserra
 fi:3Geysir
 fr:3Geyser
+hi:उष्णोत्स
```
```suggestion
hi:प्राकृतिक गरम पानी का झरना
```

proxygeek commented on data/categories.txt:
```diff
@@ -13776,6 +14044,7 @@ fa:ﻥﺎﮐﺩﻮﮐ ﻩﺎﮕﺷﻭﺮﻓ
 fi:Lasten kauppa
 fr:Magasin de puériculture|bébé
 he:םידליל תונח
+hi:बेबी सामान की दुकान
```
```suggestion
hi:बच्चों के सामान की दुकान
```

proxygeek commented on data/categories.txt:
```diff
@@ -13814,6 +14083,7 @@ fa:ﻒﯿﮐ ﻩﺎﮕﺷﻭﺮﻓ
 fi:Laukkukauppa
 fr:Magasin de sacs
 he:םיקית תונח
+hi:बैग स्टोर
```
```suggestion
hi:बैग की दुकान
```

proxygeek commented on data/categories.txt:
```diff
@@ -13928,6 +14200,7 @@ fa:ﯽﮑﯾﺮﺘﮑﻟﺍ ﻡﺯﺍﻮﻟ ﻩﺯﺎﻐﻣ
 fi:Sähköliike
 fr:Magasin de fournitures et d'appareils électriques
 he:למשח תונח
+hi:इलेक्ट्रिकल स्टोअर
```
```suggestion
hi:बिजली के सामान की दुकान
```

proxygeek reviewed (changes requested):
Have made some changes to improve readability in Hindi translations; And made some corrections. Not reviewed the types_strings.txt yet.

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -200,6 +201,7 @@
     fa = فرودگاه
     fi = Lentokenttä
     fr = Aéroport
+    hi = विमानक्षेत्र
```
```suggestion
    hi = हवाई अड्डा
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -235,6 +237,7 @@
     fa = فرودگاه
     fi = Lentokenttä
     fr = Aéroport
+    hi = अंतर्राष्ट्रीय विमानक्षेत्र
```
```suggestion
    hi = अंतर्राष्ट्रीय हवाई अड्डा
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -471,6 +477,7 @@
     fa = خودپرداز
     fi = Pankkiautomaatti
     fr = GAB
+    hi = स्वचालित गणक मशीन
```
almost no one uses or even understands the literal Hindi translation, but pretty much everyone uses the phonetic version of "ATM"
```suggestion
    hi = एटीएम
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -589,6 +599,7 @@
     fa = سَکو
     fi = Penkki
     fr = Banc
+    hi = बेंच
```
We can let it be "बेंच" as the literal translation is hardly relatable to most Hindi speakers. In fact, that's true for a lot of words like Rail, Station, etc. Such words have a defined category as well in Hindi language grammar - called "विदेशज" - meaning "Foreign", indicating the foreign origin of the words but accepted and used in Hindi

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -9648,6 +9869,7 @@
     fa = جاده
     fi = Katu
     fr = Autoroute
+    hi = हाईवे
```
```suggestion
    hi = राजमार्ग
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -12121,6 +12377,7 @@
     es = Internet
     et = Internet
     fa = وای فای
+    hi = बेतार इंटरनेट पहुंच
```
```suggestion
    hi = वाई-फाई उपलब्ध
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -12559,6 +12822,7 @@
     fa = جنگل
     fi = Metsä
     fr = Forêt
+    hi = शीतोष्ण पृथुपर्णी मिश्रित वन
```
```suggestion
    hi = मिश्रित वन
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -12612,6 +12876,7 @@
     fa = چمن زار
     fi = Ruohikko
     fr = Gazon
+    hi = मर्ग
```
```suggestion
    hi = घास
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -13930,6 +14205,7 @@
     en = Cairn
     de = Steinhaufen
     et = Karjäär
+    hi = केअर्न
```
```suggestion
    hi = स्तूप|टीला
```

j13m126 replied:
I had to drop टीला, cairns aren't searchable yet.

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -14583,6 +14868,7 @@
     fi = Ranta
     fr = Plage
     he = חוף חולי
+    hi = बालूतट
```
```suggestion
    hi = बालू तट
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -14621,6 +14907,7 @@
     fi = Hiekkainen ranta
     fr = Plage de sable
     he = חוף חולי
+    hi = बालूतट
```
```suggestion
    hi = बालू तट
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -14659,6 +14946,7 @@
     fi = Soraranta
     fr = Plage de gravier
     he = חוף חצץ
+    hi = बालूतट
```
```suggestion
    hi = बालू तट
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -15413,6 +15713,7 @@
     fi = Niitty
     fr = Prairie
     he = אָחוּ
+    hi = मर्ग
```
```suggestion
    hi = घास का मैदान
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -15604,6 +15907,7 @@
     fi = Tiheiköt
     fr = Fourré
     he = בתה
+    hi = क्षुपभूमि
```
```suggestion
    hi = झाड़ी
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -17249,6 +17578,7 @@
     fa = برج قدرت
     fi = Sähköpylväs
     fr = Poteau électrique
+    hi = उपयोगिता स्तम्भ
```
```suggestion
    hi = बिजली का खंभा
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -21426,6 +21844,7 @@
     fa = اژانس مسافرتی
     fi = Matkatoimisto
     fr = Agence de voyages
+    hi = यात्रा एजेंसी
```
Original is better and more common. The alternative is hardly used in day-to-day language

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -21871,6 +22301,7 @@
     fi = Juustokauppa
     fr = Fromagerie
     he = תוניבג תונח
+    hi = चीज़
```
```suggestion
    hi = पनीर
```

proxygeek commented on data/strings/types_strings.txt:
```diff
@@ -24227,6 +24701,7 @@
     fa = هتل
     fi = Hostelli
     fr = Auberge de jeunesse
+    hi = पथचारी आन्दोलन
```
```suggestion
    hi = शयनागार|छात्रावास|होस्टल
```

j13m126 replied:
are these synonyms? Which one of these should be seen in the app? I'll add the other two to categories.txt where search terms are stored.

proxygeek replied:
Yes, kind of synonyms. in this context, the 1st of 3 is best suited to be seen in the app.

proxygeek reviewed (changes requested):
Reviewed Hindi changes in types_strings.txt as well.

j13m126 commented:
alright, I added the suggestions by pidwid and @proxygeek and synchronized `types_strings.txt`, `categories.txt` and `categories_cuisines.txt`.

j13m126 commented:
> Folks, why do you use spreadsheets here? How can we improve this process?

I couldn't expect people on reddit to know how to review on github, so I had to provide an easy way to review.

j13m126 commented:
Do you want me to squash all commits into one or more? @biodranik @rtsisyk

vng commented:
@j13m126 Could you please rebase and merge all into 1 commit?

j13m126 commented:
@vng Could you please point me to the file where the CPP code needs changing?

Jean-BaptisteC commented:
Here https://github.com/organicmaps/organicmaps/blob/master/indexer/categories_holder.hpp

j13m126 commented:
I already added hi to `categories_holder.hpp` in [d361d32](https://github.com/organicmaps/organicmaps/pull/4266/commits/d361d323011a2fed7877a903ab40aa24b3fa089e). Any more changes that need to be made?

biodranik commented:
Please fix the conflicting files (or let us know if you need help here). Before or after the merge, Hindi should also be enabled on iOS, by adding appropriate folders for twine-generated localizations.

biodranik commented on indexer/categories_holder.hpp:
```diff
@@ -90,26 +90,27 @@ class CategoriesHolder
-      {"tr", 38},
-      {"uk", 39},
-      {"vi", 40},
+      {"hi", 21}
```
```suggestion
      {"hi", 21},
```

j13m126 replied:
done

biodranik commented:
@j13m126 please rebase your PR and carefully fix the merge conflict.

j13m126 commented:
> @j13m126 please rebase your PR and carefully fix the merge conflict.

rebased

biodranik commented on data/categories.txt:
```diff
@@ -576,6 +576,7 @@ he:כספומט
 sk:3Bankomat
 sw:Benki|fedha
 fa:خود پرداز
+hi:एटीएम|स्वचालित गणक मशीन (एटीएम)
```
1. Just to make sure: the first word matches strings.txt as it mentioned in the comment above, right? (`# First keyword should match [atm] definition in strings.txt!`)
2. categories.txt should have texts that can be typed by users. Are you sure that someone can type the second phrase here? If it can be typed, then you may add a number prefix before the longer word (usually 3, 4, or 5) to display the whole phrase as a suggestion, so instead of typing users can select the suggestion and it will be pasted into the search box.

j13m126 replied:
ad 1:  I'm pretty sure I considered it back than, but it's a long time ago already. So I will check again if the first word matches the word in strings.txt.
ad 2: I can add numbers (between 3-5) for single words depending on wordlength and my feeling of importance. For phrases idk. That could be left to natives, wdyt?

biodranik commented on data/categories.txt:
```diff
@@ -818,6 +820,7 @@ nl:3Wisselkantoor|3geld wisselen|wisselen|geld
 fi:4Rahanvaihto
 fr:3Bureau de change|3change|3devises|argent
 de:3Geldwechselstelle|Wechselstube|Geld|Geldumtausch
+hi:विनिमय कार्यालय
```
Will someone type this long phrase exactly as you wrote it here? Does it make sense to add a number before it to display as a hint? The same is related to other cases below.

biodranik reviewed (commented):
Thanks for your hard work!

Please double-check that categories.txt contains those words that users can really type in the search box, and maybe some numbers as hints there. Also, words from categories.txt should match those in types_strings.txt (we need to automate it in the future).

j13m126 commented:
Will do. 👍

j13m126 commented:
Alright, I added numbers to most words. Let me know what you think!

biodranik commented on data/categories.txt:
```diff
@@ -6851,6 +6978,10 @@ fi:Katu
 de:Straße
 hu:Utca
+fr:Rue
+de:Straße
```
Why are these values duplicated?

j13m126 replied:
I honestly don't know, but they should all be removed now.

biodranik commented on data/countries_names.txt:
```diff
@@ -22242,6 +22375,8 @@
   fr = Koweït
   de = Kuwait
+  hi = कुवैत
+  hi = कुवैत
```
?

biodranik commented on data/countries_names.txt:
```diff
@@ -22207,6 +22338,8 @@
   fr = Kiribati
   de = Kiribati
+  hi = किरिबाती
+  hi = किरिबाती
```
?

biodranik reviewed (commented):
@Saurmandal can you please check added translations?

biodranik commented:
@j13m126 sorry for the delay, we were busy with the release. Now your changes can be merged after a rebase.

j13m126 commented:
@biodranik No problem. It's rebased on latest master now. I think there's something left to do in order to enable Hindi on iOS, right? Should I create an issue for that or can you do it? I don't know what to do.

biodranik reviewed (commented):
Thanks! I've enabled iOS strings. You may accurately rebase/squash a typo and regeneration commits to clean up the history.

Next step is to test if it works on real Android and iOS devices or simulators:
- Interface translations
- Search in Hindi
- Search categories (categories.txt entries should match types_strings.txt)
- TTS sounds

j13m126 commented:
Rebased on current master and squashed those two commits to 'Initial Hindi translation' commit. 
Ad testing on Android: Running generate_localizations.sh doesn't create a values-hi folder with strings.xml yet. Any ideas why?
Ad testing on iOS: I don't a Mac so I don't think I can build it.

biodranik commented:
1. Looks like there are many missing translations for types (e.g. Industrial Area, National Park...) and the app's interface. Were they lost accidentally, or it is just a work in progress?
2. The search does not work. At least `[[Search categories]]` section in strings.txt should be translated and match categories.txt translations.
3. Does it work on Android?

j13m126 commented:
> 1. Looks like there are many missing translations for types (e.g. Industrial Area, National Park...) and the app's interface. Were they lost accidentally, or it is just a work in progress?
> 
>     2. The search does not work. At least `[[Search categories]]` section in strings.txt should be translated and match categories.txt translations.
> 
>     3. Does it work on Android?
> 
>     4. The last commit misses DCO.

Alright, I added a few interface strings, translated search categories and made sure they match categories.txt translations.
I could copy a bunch of interface strings from [bing maps](https://www.bing.com/maps/?cp=48.218725%7e16.342163&lvl=11.0&setlang=hi) by comparing the hindi and english version. I double checked every word with google translate in order not to copy the wrong word. 

I tested in on my Pixel 6, running Android 14. Switching phones and also the apps language. I could also enable and hear TTS in hindi. But only on my phone, not on my samsung s6 lite tablet (Android 13). Hindi wasn't an option there.

Countries etc. in the map download section are currently not showing the hindi names, even though I provided some in countries_names.txt.

If you need any more important interface strings, let me know, otherwise I'd like to leave it as is.

j13m126 commented:
rebased

Jean-BaptisteC commented:
Rebase is not enough :(
Can you rebase and regenerate strings?

Jean-BaptisteC commented:
@biodranik ready to merge
@j13m126 thanks for your patience :)

biodranik commented:
@vng 

```
Running categories_test.cpp::CategoriesIndex_AllCategories
Number of nodes in the CategoriesIndex trie: 662233
FAILED
indexer_tests/categories_test.cpp:318 TEST(index.GetNumTrieNodes() < 650000) 662233 650000 
Test took 8475 ms
```

biodranik reviewed (approved):
Thank you for such a great contribution!

vng reviewed (commented):
Are we sure that regenerated strings are ok here?
- I don't see android strings file
- iOS localization file has a lot of English strings

biodranik commented:
@Jean-BaptisteC can you please check if Hindi is working on Android, and also check if each of the search categories works (categories.txt matches types_strings.txt)?

Jean-BaptisteC commented:
Translations and searches in Hindi works on Android
